Instagram-SERVER/src/main/java/com/sopt/instagram/InstagramSERVER/post/domain/Post.java

```diff
@@ -0,0 +1,25 @@
+package com.sopt.instagram.InstagramSERVER.post.domain;
+
+import com.sopt.instagram.InstagramSERVER.member.domain.Member;
+import lombok.*;
+
+import javax.persistence.*;
+
+@Builder
+@Getter
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Post {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "post_id")
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "member_id", nullable = false)
+    private Member member;
+
+    private String content;
+}

```

Comment:
p3;

위에 리뷰 달아드린 @Builder를 아래와 같이 생성자 만들어서 거기에 직접 달아주는 것이 좋을 것 같아요 !

```suggestion
     @Builder
     public Post (Member member, String content) {
         this.member = member;
         this.content = content;
     }

}

```